client/package.json

```diff
@@ -17,7 +17,8 @@
     "react-dom": "^16.13.1",
     "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.0"
+    "react-scripts": "^4.0.0",
+    "react-test-renderer": "^17.0.1"
```

Comment:
I don't think this is used. If that's true, remove it and rerun `yarn lock`

Reply:
I think this one was added automatically. I just did a quick reseach about this dependency, and I think it will be okay if I just simply remove this. So should I just run "yarn remove react-test-renderer" in terminal, it will remove this denpendency and update "yarn lock", correct?

Reply:
yes you can try it